src/open_inwoner/openzaak/tests/test_migrations.py

```diff
@@ -172,3 +172,32 @@ def test_migration_0051_to_0052_raises_with_descriptive_exception_message(self):
             " This field is now required: please manually update all the affected rows or re-sync"
             " your ZGW objects to ensure the field is included.",
         )
+
+
+class TestAllServicesExceptFormsRequiredForZGWApiGroup(TestFailingMigrations):
```

Comment:
Since this is testing the migration, perhaps `ZGWAPIGroupRequirementsFailingMigrationTest` or something like it?

Also, could use a test for a migration that succeeds.